site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -0,0 +1,199 @@
+Elements with `interesttarget` will - when hovered, long pressed, or focussed -
+dispatch an `InterestEvent` on the element referenced by `interesttarget`, with
+some default behaviours.
+
+### Terms
+
+- Interest/Shows Interest: The action of _Interest_ refers to the user "landing"
+  on an element without invoking it, using a HID. If the HID is a mouse, this
+  would be a `hover` event. If the HID is a touch screen, this would be a long
+  press on the element using a stylus or finger, if the HID is a keyboard, this
+  would be when the element is focussed. For other HIDs such as eye tracking or
+  pedals or game controllers, the equivalent "focus or hover" action is used to
+  take interest on the element.
+- Loses/Lost Interest: The action of _Loses Interest_ refers to the user "moving
+  away" from an element, or its _interestee_, using a HID; in other words
+  interest must be on a different element that is neither. Elements can only
+  _Lose Interest_ if they are in the state of Showing Interest. If the HID is a
+  mouse, this would be a `mouseout` event. If the HID is a touch screen, this
+  would be long pressing outside of the elements bounds. If the HID is a
+  keyboard, this would be moving focus away from the element. For other HIDs
+  such as eye tracking or game controllers, the equivalent "focusout or
+  mouseout" action is used to _Lose Interest_ on the element.
+- Interestee: An element which is referenced to by an Interest element, via the
+  `interesttarget` attribute.
+
+## Proposed Plan
+
+In the style of `invoketarget`, this document proposes we add `interesttarget`
+as an available attribute to `interesttarget` attribute to `<button>`, `<a>`,
+`<area>` and `<input>` elements.
+
+```webidl
+interface mixin InvokerElement {
+  [CEReactions] attribute Element? invokeTargetElement;
+  [CEReactions] attribute DOMString invokeAction;
+};
+HTMLButtonElement extends InvokerElement
+HTMLInputElement extends InvokerElement
+```
+
+The `interesttarget` value should be an IDREF pointing to an element within the
+document. `.interestTargetElement` also exists on the element to imperatively
+assign a node to be the invoker target, allowing for cross-root invokers (in
+some cases, see [the popovertarget attr-asociated element
+steps](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#attr-associated-element)
+for more).
+
+Elements with `interesttarget` set will dispatch an `InterestEvent` on the
+_Interestee_ (the element referenced by `interesttarget`) when the element
+_Shows Interest_ or _Loses Interest_. When the element _Shows Interest_ the
+event type will be `'interest'`. If the element has _Shown Interest_, and
+interest moves away from both the _Interest Element_ and the _Interestee_, then
+the element _Loses Interest_ and an `InterestEvent` with the type of
+`'loseinterest'` will be dispatched on the _Interestee_. The event also contains
+a `relatedTarget` property that will reference the _Interest_ element.
```

Comment:
Did this property get renamed on the invoke event? If so rename accordingly here